network-api/networkapi/wagtailpages/image_formats.py

```diff
@@ -0,0 +1,8 @@
+from wagtail.images.formats import Format, register_image_format, unregister_image_format
+
+# We can't override existing image format,
+# so we are unregistering it and then re-registering it with new values instead.
+# Set it to 2280px because 1140px is the max width of Bootstrap's container.
+# Then we multiply that number by 2 so the image can render well on retina.
+unregister_image_format('fullwidth')
+register_image_format(Format('fullwidth', 'Full width', 'richtext-image full-width', 'width-2280'))
```

Comment:
Or is it less risky if we add a new option instead of modifying the existing `full width` option? Something like `full width high quality`?

Reply:
As far as I'm aware this should be fine. But there may be cases where we don't want an image to be 2280px wide (like if someone has a 1000px image). If the uploaded image isn't at least 2280px wide it'll stretch the image to make it wider which will blur it. 

I like the idea of adding a new high quality image format so that way there's the standard way and the new high def way. This wouldrequire a little staff education.

Reply:
do we have any control over the resize algorithm? If not, having a way to say "don't upscale" would be good.

Reply:
@Pomax I think not putting any resizing option would render the image in its original size (e.g., no `width-*`, `fill-*` etc)

@KalobTaulien I had a similar concern when working on this, and I struggled to find a good UX solution to it. Our current image alignment/resizing options on Wagtail will resize images by default:
```py
register_image_format(Format('fullwidth', _('Full width'), 'richtext-image full-width', 'width-800'))
register_image_format(Format('left', _('Left-aligned'), 'richtext-image left', 'width-500'))
register_image_format(Format('right', _('Right-aligned'), 'richtext-image right', 'width-500))
```

How do we make it more explicitly so that users can know images will be resized automatically? Should we update the image option label so it says, for example, `Right-aligned (will automatically be resized to 500px wide)` instead of `Right-aligned`?

Reply:
I think this comes down to just educating staff on the CMS.